python_scripts/xx_basic_preprocessing.py

```diff
@@ -0,0 +1,352 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     formats: notebooks//ipynb,markdown_files//md,python_scripts//py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.2'
+#       jupytext_version: 1.2.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Introduction to scikit-learn: basic preprocessing for basic model fitting
+#
+# In this lecture note, we will aim at introducing:
+# * the difference between numerical and categorical variables;
+# * the importance of scaling numerical variables;
+# * the way to encode categorical variables;
+# * combine different preprocessing on different type of data;
+# * evaluate the performance of a model via cross-validation.
+#
+# %% [markdown]
+# ## Introduce the dataset
+#
+# To this aim, we will use data from the 1985 "Current Population Survey"
+# (CPS). The goal with this data is to regress wages from heterogeneous data
+# such as age, experience, education, family information, etc.
+#
+# Let's first load the data located in the `datasets` folder.
+
+# %% {"deletable": true, "editable": true}
```

Comment:
Hmmm out of interest, does that come from jupytext? I did not remember this I have to say (maybe this happened in a recent version). I feel this is a bit ugly and goes against the idea that the `.py` file is easy to write ... I am hoping that it is fine not to mention anything and that it gets automatically populated at one point during the sync.

Reply:
I have the same filling. I used it because it was a marker given when you translate a notebook to `py`. If we can remove it and this is working I am all for it.